Nos/Views/DiscoverView.swift

```diff
@@ -169,6 +172,20 @@ struct DiscoverView: View {
             .animation(.easeInOut, value: columns)
             .task { 
                 updatePredicate()
+
+                if concecutiveTapsCancellable == nil {
```

Comment:
Everything inside `.task` is guaranteed to only run once, so I think we could get rid of the `if concecutiveTapsCancellable == nil {` check.

Reply:
I don't think so, did that change in the API? .task did execute more than once for me, everytime the view appears

Reply:
Yep you are right. I put some breakpoints in and it seems like .task is called anytime a view's state is mutated!

Reply:
Cool, I'm keeping the cancellable and checking the nullness then